RAKE/RAKE.py

```diff
@@ -4,8 +4,11 @@
 # Automatic keyword extraction from indi-vidual documents.
 # In M. W. Berry and J. Kogan (Eds.), Text Mining: Applications and Theory.unknown: John Wiley and Sons, Ltd.
 
+from __future__ import absolute_import
 import re
 import operator
+import six
+from six.moves import range
```

Comment:
That makes my nervous for production things. There's a cross version vectorized version of generator functions we could use instead right?